test/metabase/db/schema_migrations_test/impl.clj

```diff
@@ -109,6 +109,8 @@
             (parse-double (format "%d.%d" (* (parse-long major-version) 100) unix-timestamp))))
         (throw (ex-info (format "Invalid migration ID: %s" id) {:id id})))))
 
+(alter-var-root #'migration->number memoize)
```

Comment:
If `migration->number` should be memoized you should just do it when you define it in the first place instead of using `alter-var-root`. Is parsing migration numbers really that much of a performance hit in tests tho?